src/extension.ts

```diff
@@ -11,107 +12,55 @@ interface Params {
     file: string | null
 }
 
-interface SentryProject {
-    projectId: string
-    lineMatches: RegExp[]
-}
-
 const DECORATION_TYPE = sourcegraph.app.createDecorationType()
 const SETTINGSCONFIG = resolveSettings(sourcegraph.configuration.get<Settings>().value)
+const SENTRYORGANIZATION = SETTINGSCONFIG['sentry.organization']
+
+const COMMON_ERRORLOG_PATTERNS = [
+    /throw new Error+\(['"]([^'"]+)['"]\)/gi,
+    /console\.[^'"`]+\(['"`]([^'"`]+)['"`]\)/gi,
+    /log\.[^'"]+\(['"]([^'"]+)['"]\)/gi,
```

Comment:
Can you give a real-world example of this? It seems to match strings like `log."foo"`.

Reply:
It shouldn't match `log."foo"` but rather patterns like `log.Print("foo")` or `log.Printf("bar")`
![image](https://user-images.githubusercontent.com/9110008/55369654-4af21d00-54ac-11e9-9924-e052c1818a44.png)

Reply:
Ok, I'll limit it to the known `log.Print*()` options.